### PR TITLE
More validations

### PR DESCRIPTION
**Pull request by foFox**

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 class Category < ActiveRecord::Base
   attr_accessible :name
+  validates :name, :presence => true
+  validates_uniqueness_of :name
```
Comment (tsov) [inline app/models/category.rb]: can you use `validates :name, :presence => true, :uniqueness => true`

Comment (alankehoe) [inline app/models/category.rb]: Go to work haha

Alan Kehoe

File: app/models/event.rb [hunk 2]
```diff
@@ -11,5 +11,6 @@ class Event < ActiveRecord::Base
   validates :description, :name, :start_time, :end_time, :place, :presence => true
   validates :name, :length => { :minimum => 5, :maximum => 256 }
   validates :description, :length => { :minimum => 5, :maximum => 1024 }
+  validates :visibility_type, :inclusion => { :in => %w(public private certified)}
```
Comment (tsov) [inline app/models/event.rb]: This is also a good way to do it, but maybe we should you enums? https://github.com/lwe/simple_enum

File: app/models/category.rb (hunk 1 above)
Comment (tsov) [inline app/models/category.rb]: haha, shut up alan! :D

**Comment by tsov** [comment]
Didn't we agree that the event will inherit the type from the parent place?

File: app/models/event.rb [hunk 3]
```diff
@@ -12,4 +12,9 @@ class Event < ActiveRecord::Base
   validates :name, :length => { :minimum => 5, :maximum => 256 }
   validates :description, :length => { :minimum => 5, :maximum => 1024 }
   validates_with EventhubApi::Validator::Event
+
+  def visibility_type
+    self.parent.visibility_type
+  end
```
Comment (tsov) [inline app/models/event.rb:18]: I think this might be a quick solution for the problem (As you would say in business :D)

Comment (foFox) [inline app/models/event.rb:18]: Thats a really nice way of doing it.

Comment (tsov) [inline app/models/event.rb:18]: Sweet! Lets merge this then

**Pull request merged**